src/components/ChatList/CommunityChannelItem.tsx

```diff
@@ -11,7 +11,7 @@ import {channelItemStyles as styles} from './ChannelItem.style';
 
 const CommunityChannelItem = (props: ChannelItemProps) => {
   const {channel: community, onChannelPressed} = props;
-  const channelType = 'COMMUNITY';
+  const channelType = community.channelType || 'COMMUNITY';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change made here is good as it allows for different types of channels, not just 'COMMUNITY'. However, it would be better to define the default channel type ('COMMUNITY') as a constant at the top of the file or in a separate constants file. This will make the code more maintainable and less prone to errors due to typos.
```diff
- const channelType = community.channelType || 'COMMUNITY';
+ const DEFAULT_CHANNEL_TYPE = 'COMMUNITY';
+ const channelType = community.channelType || DEFAULT_CHANNEL_TYPE;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->